open-company/open-company-web
# Quickstart Guide

Card: https://trello.com/c/oYmbNoPJ

Requires:
- [Storage](https://github.com/open-company/open-company-storage/pull/187)
- [Auth](https://github.com/open-company/open-company-auth/pull/78)

To test:
- onboard a new email user/org
- [x] do you see the create first post tooltip when you land in AP? good
- add your first post
- [x] did you see the QSG open up? Good
- [x] do you see a tooltip pointing at WST? Good
- [x] create a post is already checked? Good
- complete the QSG steps
- [x] add a section
- [x] did you get the tooltip pointing to the section COG? Good
- [x] verify email
- [x] add profile photo
- [x] add company logo
- [x] invite team
- [x] see a sample digest
- [x] checkout reminders
- [x] progress bar is 100%? Good
- [x] now watch the 60 seconds video, did it work? Good
- [x] now dismiss the QSG view
- [x] did you get a tooltip saying that you completed it? Good
- refresh
- [x] QSG still closed? Good
- open QSG manually from the menu
- refresh
- [x] QSG still open? Good
- dismiss QSG
- onboard an invited admin now
- [x] do you see the first welcome tooltip? Good
- add a post
- [x] do you NOT see the QSG open now? Good
- open the QSG
- refresh
- [x] do you see the QSG? Good
- complete the items you see
- [x] does the QSG progress bar is at 100%? Good
- refresh
- [x] QSG still open and at 100%? Good
- dismiss it
- refresh
- [x] qsg still dismissed? Good
- now onboard an invited contrib
- [x] do you see a welcome tooltip? Good
- create a post
- [x] do you NOT see QSG? Good
- now check the dropdown menu
- [x] do you NOT see QSG? Good
- now onboard an invited contrib
- now check the dropdown menu
- [x] do you NOT see QSG? Good
